cypress/e2e/04_opening_and_closing_node_modals.cy.ts

```diff
@@ -0,0 +1,61 @@
+/// <reference types="cypress" />
+require('cypress-xpath')
+describe("Opening and closing node modules", () => {
+  beforeEach(() => {
+    cy.viewport(1920, 1200)
+    cy.eyesOpen({
+      appName: "studio",
+      testName: Cypress.currentTest.title,
+    });
+  });
+
+
+  it("should open flojoy studio's main page", () => {
+    cy.visit("/").wait(1000);
+
+    cy.eyesCheckWindow({
+      tag: "dark flow page",
+      target: "window",
+      fully: true,
+    });
+    // Click on added container LINSPACE
+    cy.xpath("//div[contains(text(), 'LINSPACE')]").click();
```

Comment:
Instead of checking for only two nodes explicitly, can we have an array of nodes and loop through each node and do the same action for them all?